spec/index.bs

```diff
@@ -569,6 +561,63 @@ enum GPUBindingType {
 };
 </script>
 
+  * {{GPUBindGroupLayoutBinding/textureDimension}}, {{GPUBindGroupLayoutBinding/multisampled}}:
+    The dimensions and multi-sampling properties of texture bindings.
+
+    Note: This allows Metal-based implementations to back the respective bind
+    groups with `MTLArgumentBuffer` objects that are more efficient to bind at
+    run-time.
+
+  * {{GPUBindGroupLayoutBinding/dynamic}}:
```

Comment:
We should probably rename this to something like `dynamicOffset`. Subsequently, it will be clearer to always say "dynamic offset binding" instead of "dynamic binding".

Reply:
`dynamicOffset` sounds like it should be a number; maybe `dynamicallyOffset`? I still prefer `dynamic` for a bool.

Reply:
Agreed. `hasDynamicOffset`?

Reply:
Renamed to `hasDynamicOffsets`.